### ISSUE TITLE
Connections Tracking Issue

### ISSUE THREAD

**Issue by JakeNeyer** [polytomic/terraform-provider-polytomic#52]
The following connections types need to be added to the terraform provider:

- [x] Ascend
 - [x] Affinity
 - [x] Airtable
 - [x] Amplitude
 - [x] API
 - [x] Asana
 - [x] AWS Athena
 - [x] Bing Ads
 - [x] ChargeBee
 - [x] CloudSQL
 - [x] CosmosDB
 - [x] CSV
 - [x] CustomerIO
 - [x] Databricks
 - [x] Delighted
 - [x] Dialpad
 - [x] Fresh Desk
 - [x] Fullstory
 - [x] Github
 - [x] Google Ads
 - [x] Harmonic
 - [x] Intercom
 - [x] Iterable
 - [x] Klaviyo
 - [x] Kustomer
 - [x] Linear
 - [x] LinkedIn Ads
 - [x] Lob
 - [x] Marketo
 - [x] MongoDB
 - [x] Mysql
 - [x] Netsuite
 - [ ] Pardot
 - [x] Pipedrive
 - [x] Redshift
 - [x] Salesforce
 - [x] Segment
 - [x] Shipbob
 - [x] Shopify
 - [x] Smartsheet
 - [x] Statsig
 - [x] Stripe
 - [x] Survicate
 - [x] Synapase
 - [x] Uservoice
 - [x] Vanilla
 - [x] Webhooks
 - [x] Zendesk

**Comment by leonkozlowski** [comment]
Hi @JakeNeyer curious to whether or not salesforce will be added to this list?

**Comment by leonkozlowski** [comment]
> > Hi @JakeNeyer curious to whether or not salesforce will be added to this list?
> 
> Hey @leonkozlowski! We currently only plan to support [salesforce as a datasource](https://registry.terraform.io/providers/polytomic/polytomic/latest/docs/data-sources/salesforce_connection). Since the salesforce connection uses an oauth flow which requires user interaction, there's not a great way to support it as a resource at this time.

I see, @JakeNeyer, does this mean, using the importer + using a data block to reference a salesforce source is the best course of action to move polytomic connection/model/sync definitions to terraform?

**Comment by JakeNeyer** [comment]
@leonkozlowski yup! The importer will actually output the data blocks for connections that are only available as datasources as well.

**Comment by nyergler** [comment]
We've moved to generating all connections using the JSONSchema and the generated SDK; closing.

**Comment by Mark-Lasfar** [comment]
How can I integrate with Shipbob? I created the special programming for integration, but when I ran the test, nothing was executed inside the Shipbob control panel. I ran a test on adding an order, fetching products, and fetching information, but in those cases of testing, an error appears. I created an account inside Shipbob and added the account information and generated the token API pat, but I did not get any results, which makes me doubt the key I created and how I can give the necessary permissions to this key. Note: I am doing direct integration without using any e-commerce platforms. I have my own store that I designed, but to solve the problem
